### PR TITLE
Build fix for Java 11

### PR DESCRIPTION
**Pull request by ya-mouse**
Minor fixes to make the Jep compiled with Java 11.

### CODE REVIEW

**Review by bsteffensmeier — APPROVED** [review]

**Comment by ya-mouse** [comment]
Who might merge this PR once approved?
Thanks!

**Comment by bsteffensmeier** [comment]
> Who might merge this PR once approved?
> Thanks!

I'm waiting a few days to see if @ndjensen wants to review it. If he doesn't review this week then I will merge it.

**Review by ndjensen — APPROVED** [review]

**Pull request merged**
